routes/web.php

```diff
@@ -99,8 +99,8 @@
 
     Route::prefix('settings')->namespace('Settings')->group(function () {
         Route::get('/', 'SettingController@index')->name('settings.index');
-        Route::get('/employee-earning-threshold', 'ProfitabilityController@index')->name('settings.employee-earning-threshold');
-        Route::post('/employee-earning-threshold', 'ProfitabilityController@update')->name('settings.update-employee-earning-threshold');
+        Route::get('/config-variables', 'ConfigurationController@index')->name('settings.config-variables');
+        Route::post('/config-variables', 'ConfigurationController@update')->name('settings.update-config-variables');
```

Comment:
```suggestion
        Route::resource('config-variables', 'ConfigurationController')->only(['index', 'update']);
```

// config-variables.index
// config-variables.update